### ISSUE TITLE
Improve how event format is stored

### ISSUE THREAD

**Issue by 0916dhkim** [EOT-Event-Organizer-s-Toolbox/eot-codebase#174]
Currently we have `CommunityEvent.inPersonEvent` and `CommunityEvent.onlineEvent` for storing the event format. It is not the best way to save this information because it doesn't make sense to have an event that is both in-person & online, or neither in-person nor online.

Let's improve this using prisma [enum](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-enums).

**Comment by anjolaoluwaakindipe** [comment]
We could make it an enum and call it attendance type and have the following options: Online, InPerson, OnlineAndInPerson. This could solve the issue of having an event that is neither in person or online

**Comment by 0916dhkim** [comment]
Yep I like the having an option for the online/in-person hybrid 👍
